json_reader_test.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_jsonReader_Unmarshal(t *testing.T) {
```

Comment:
Achte darauf nichts unwichtiges zu testen. json.Unmarshal funktioniert weil es in der stdlib ist. Bei tests geht es darum DEINEN code zu testen nicht den von anderen!